src/sentry/integrations/slack/utils.py

```diff
@@ -351,3 +367,49 @@ def send_confirmation(
                 "team_id": integration.external_id,
             },
         )
+
+
+def get_referrer_qstring(notification: BaseNotification) -> str:
+    return "?referrer=" + re.sub("Notification$", "Slack", notification.__class__.__name__)
+
+
+def get_settings_url(notification: BaseNotification) -> str:
+    if isinstance(notification, ReleaseActivityNotification):
+        fine_tuning = "deploy/"
```

Comment:
I think we should put the `fine_tuning_key` as a property in the `ActivityNotification` and override it.